README.md

```diff
@@ -1,11 +1,7 @@
 
 OG-USA
 =========
-OG-USA is an overlapping-generations (OG) model of the economy of the United
-States (USA) that allows for dynamic general equilibrium analysis of federal
-tax policy.  The model outputs focus on the change in macroeconomics aggregates
-(GDP, investment, consumption), wages, interest rates, and the stream of tax
-revenues changes over time.
+OG-USA is an overlapping-generations (OG) model of the economy of the United States (USA) that allows for dynamic general equilibrium analysis of federal tax policy. The model outputs focus on the change in macroeconomics aggregates (GDP, investment, consumption), wages, interest rates, and the stream of tax revenues changes over time. Careful documentation of the model, its output, and solution method is available in this repository at [https://github.com/rickecon/OG-USA/blob/doc/documentation/OGUSAdoc.pdf](https://github.com/rickecon/OG-USA/blob/doc/documentation/OGUSAdoc.pdf) and is regularly updated.

```

Comment:
@rickecon To correct some errors from prior versions, I think the second sentence should be: "The model output focuses changes in macroeconomic aggregates (GDP, investment, consumption), wages, interest rates, and the stream of tax revenues over time."

Also, should the final sentence:
* Say "... and the solution method are available"?
* Refer to "this repository" (sounds like open-source-economics/OG-USA repo)?  Maybe just say "are available here" and have "here" be hyperlinked to the pdf from the documentation repo?